<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTimeEdit" name="start_time_edit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="time">
    <time>
     <hour>9</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QTimeEdit" name="end_time_edit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="time">
    <time>
     <hour>15</hour>
     <minute>30</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>START :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>END :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="statusTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
